src/api/procedures/__tests__/consumeJoinIdentityAuthorization.ts

```diff
@@ -9,13 +9,13 @@ import {
   prepareStorage,
   Storage,
 } from '~/api/procedures/consumeJoinIdentityAuthorization';
-import { Account, AuthorizationRequest, Context, Identity } from '~/internal';
+import { Account, AuthorizationRequest, Context, Identity, PolymeshError } from '~/internal';
 import { dsMockUtils, entityMockUtils, procedureMockUtils } from '~/testUtils/mocks';
 import { Mocked } from '~/testUtils/types';
-import { Authorization, AuthorizationType, Signer, TxTags } from '~/types';
+import { Authorization, AuthorizationType, ErrorCode, Signer, TxTags } from '~/types';
 import * as utilsConversionModule from '~/utils/conversion';
 
-describe('consumeJoinSignerAuthorization procedure', () => {
+describe('consumeJoinIdentityAuthorization procedure', () => {
```

Comment:
Can you please also change the same in `consumeAddMultiSigSignerAuthorization` test as well?